jenkins-pipelines/manager/enterprise-ami-manager-sanity.jenkinsfile

```diff
@@ -0,0 +1,22 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+managerPipeline(
+    backend: 'aws',
+    ip_ssh_connections: 'public',
+    region: '''["us-east-1", "us-west-2"]''',
+
+    scylla_version: '2021.1',
+
+    test_name: 'mgmt_cli_test.MgmtCliTest.test_manager_sanity',
+    test_config: 'test-cases/manager/manager-regression-multiDC-set-distro.yaml',
+
+    timeout: [time: 240, unit: 'MINUTES'],
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+
+    downstream_jobs_to_run: 'enterprise-ami-upgrade-test'
```

Comment:
shouldn't it be the same name as the jenkinsfile? (enterprise-ami-manager-upgrade.jenkinsfile)?

Reply:
I think this is a reminder that manager jobs are created by hand, and not via a hydra command (like in scylla releases). 
so the names might vary based on whom created the jobs.

also the job name is the jenkinsfile basename + `-test`

the automated code path would have created a job named: `enterprise-ami-manager-upgrade-test`

Reply:
That is true